### PR TITLE
MORE VERBS! [MODULAR] 

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## About The Pull Request
Adds the verbs LOOC.

Adds the emote subtle and subtler. 

Tweaks WHO to make more sense.

## Why It's Good For The Game

Muh ERP.

## Changelog
:cl:
add: Added LOOC, subtle, and subtler. Tweaked WHO too.
/:cl:

### CODE REVIEW

**Comment by Azarak** [comment]
Instead of using "chat_toggles", add a new variable called "skyrat_toggles" for our bitflags, to avoid conflicts, especially since that is gonna get saved and loaded. Have LOOC be enabled by default too.

**Comment by Azarak** [comment]
rlooc style class doesn't exist

**Comment by Gandalf2k15** [comment]
fixed above issues.

**Comment by Azarak** [comment]
make TOGGLES_DEFAULT_CHAT_SKYRAT be TOGGLES_DEFAULT_SKYRAT, as I reckon it will be for all sorts of toggles related to us. Also, it needs to save/load

File: modular_skyrat/modules/verbs/code/modules/client/preferences_toggles.dm [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+TOGGLE_CHECKBOX(/datum/verbs/menu/Settings, listen_looc)()
```
Comment (Azarak) [inline modular_skyrat/modules/verbs/code/modules/client/preferences_toggles.dm]:
```suggestion
/client/verb/listen_looc()
```
TG doesn't really use checkboxes

File: modular_skyrat/modules/verbs/code/modules/client/preferences_toggles.dm [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+	set name = "Show/Hide LOOC"
+	set category = "Preferences"
+	set desc = "Toggles seeing LocalOutOfCharacter chat"
+	usr.client.prefs.skyrat_toggles ^= CHAT_LOOC
+	usr.client.prefs.save_preferences()
+	to_chat(usr, "You will [(usr.client.prefs.skyrat_toggles & CHAT_LOOC) ? "now" : "no longer"] see messages on the LOOC channel.")
+	SSblackbox.record_feedback("nested tally", "preferences_verb", 1, list("Toggle Seeing LOOC", "[usr.client.prefs.skyrat_toggles & CHAT_LOOC ? "Enabled" : "Disabled"]"))
+/datum/verbs/menu/Settings/listen_ooc/Get_checked(client/C)
+	return C.prefs.skyrat_toggles & CHAT_LOOC
```
Comment (Azarak) [inline modular_skyrat/modules/verbs/code/modules/client/preferences_toggles.dm]:
```suggestion
```

**Pull request merged**